### ISSUE TITLE
Question regarding Labor Hour Not-to-Exceed

### ISSUE THREAD

**Issue by linda1902** [18F/s70-disa-eapp#2]
# Question/Comment on National Background Investigation System (NBIS), eApp 
Hours provided in RFQ Document - please clarify
## Name and affiliation
Linda Schmidt Softrams

## Section of RFQ documents
RFQ - Section 1.2.1

## Question/Comment
Section 1.2.1 BASE PERIOD: LABOR HOUR CLIN - are the hours indicated, 3396 for the base period and 1998 for the option periods the maximum hours each contractor is supposed to bid for services?

**Comment by michaelccata** [comment]
Yes, offerors may propose as many labor categories as necessary with their respective rates but shall not exceed the indicated hours. Offerors shall quote using the indicated hours for each period of performance.

**Comment by 18Facq** [comment]
Yes, offerors may propose as many labor categories as necessary with their respective rates but shall not exceed the indicated hours. Offerors shall quote using the indicated hours for each period of performance.
